src/index.html

```diff
@@ -11,7 +11,96 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input class="element" type="text" name="surame"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field">Name:
+          <input class="element" type="text" name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field">How old are You?
+          <input class="element" type="number" name="age"
+          min="1" max="100" value="12"
+          >
+        </label>
+        <label class="form-field">Full date of birth:
+          <input class="element" type="date" name="date-of-birth">
+        </label>
+        <label class="form-field">I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">E-mail:
+          <input class="element" type="email" name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">Password:
+          <input type="password" name="password"
+          minlength="6" maxlength="9" required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">Do you love cats? 
+          <label>
+            <input class="element" type="radio" name="love-cats"
```

Comment:
```suggestion
            <input
              class="element"
              type="radio"
              name="love-cats"
```

fix formatting everywhere